docs/project-structure/genezio-configuration-file.md

```diff
@@ -629,6 +629,95 @@ frontend:
       - npm run dev
 ```
 
+## container
+
+The Docker container configuration. This field can be omitted if the project is not containerized.
+
+- path: `string` **Required**
+
+  The path to the Dockerfile. It is relative to the `genezio.yaml` file.
+
+- environment: `Object` **Optional**
+
+  The environment variables that will be set for the server inside the Docker container.
+
+  You can use expression to define the environment variables. Check the [Usage](#expressions) section for more information.
+
+- timeout: `number` **Optional**
```

Comment:
Are these implemented for docker? Are they passed correctly?

Reply:
Yes - I checked the CLI code and it looks okay.